RecoMuon/MuonIdentification/plugins/MuonIdProducer.cc

```diff
@@ -452,6 +457,24 @@ void MuonIdProducer::produce(edm::Event& iEvent, const edm::EventSetup& iSetup)
 
    init(iEvent, iSetup);
 
+   std::auto_ptr<reco::MuonTimeExtraMap> muonTimeMap(new reco::MuonTimeExtraMap());
+   reco::MuonTimeExtraMap::Filler filler(*muonTimeMap);
+   std::auto_ptr<reco::MuonTimeExtraMap> muonTimeMapDT(new reco::MuonTimeExtraMap());
+   reco::MuonTimeExtraMap::Filler fillerDT(*muonTimeMapDT);
+   std::auto_ptr<reco::MuonTimeExtraMap> muonTimeMapCSC(new reco::MuonTimeExtraMap());
+   reco::MuonTimeExtraMap::Filler fillerCSC(*muonTimeMapCSC);
+
+   std::auto_ptr<reco::IsoDepositMap> trackDepMap(new reco::IsoDepositMap());
+   reco::IsoDepositMap::Filler trackDepFiller(*trackDepMap);
+   std::auto_ptr<reco::IsoDepositMap> ecalDepMap(new reco::IsoDepositMap());
+   reco::IsoDepositMap::Filler ecalDepFiller(*ecalDepMap);
+   std::auto_ptr<reco::IsoDepositMap> hcalDepMap(new reco::IsoDepositMap());
+   reco::IsoDepositMap::Filler hcalDepFiller(*hcalDepMap);
+   std::auto_ptr<reco::IsoDepositMap> hoDepMap(new reco::IsoDepositMap());
+   reco::IsoDepositMap::Filler hoDepFiller(*hoDepMap);
+   std::auto_ptr<reco::IsoDepositMap> jetDepMap(new reco::IsoDepositMap());
+   reco::IsoDepositMap::Filler jetDepFiller(*jetDepMap);
```

Comment:
It looks like I was editing an older version of the muonIdProducer that does not use "fillmap", like the modern one in 80X does.  It was not our intention to alter that functionality, I assume I should revert this section as well as this one: https://github.com/cms-sw/cmssw/pull/13308/files#diff-89dc8f122700a0c3b857022fe730fd42L728

Should I go ahead and revert those?